components/NavigationContainer/NavigationContainer.style.js

```diff
@@ -0,0 +1,12 @@
+import { styled } from "styled-components";
+
+export const StyledNavigationContainer = styled.div`
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+
+  ${(props) => (props.right ? "right: 0px" : "")};
+  ${(props) => (props.left ? "left: 0px" : "")};
+  ${(props) => (props.top ? "top: 60px" : "")};
+  ${(props) => (props.bottom ? "bottom: 60px" : "")};
+`;
```

Comment:
positioning your navigation absolute might not be the best choice, you could review the [CSS Positioning - Handout](https://github.com/neuefische/bo-web-23-3/blob/main/sessions/css-positioning/css-positioning.md) for further inspiration.

Passing props is a good choice if you want variations, but are you really using this component in multiple places?

Reply:
you are right - at this point i am only using it at one point. i changed it because i also want to use it for other items

Reply:
I am using it in multiple places and in future features i am going to reuse it for the navigation of the user